src/definition/populate_test.go

```diff
@@ -80,6 +80,76 @@ func metricsNamingManipulator(ms metric.MetricSet, entity sdk.Entity, clusterNam
 	return ms.SetMetric("displayName", entity.Name, metric.ATTRIBUTE)
 }
 
+func TestIntegrationProtocol2PopulateFunc_AddingNewMetricSetToEntity(t *testing.T) {
```

Comment:
A bit out of local conventions, but perhaps test name could be more descriptive, e.g.
```suggestion
func Test_IntegrationProtocol2Populate_generates_single_metric_set_for_single_entity_for_metrics_from_multiple_groups(t *testing.T) {
```

Reply:
nice, I will use that name. but i think the rest of the tests has the convention `TestNameOfFunc_NameOfTest` I will follow that to be consistent with the rest.